README.md

```diff
@@ -204,3 +212,351 @@ jobs:
+| project_version  | Version of the project                        | False    |               |
+| project_name     | Name of the project                           | False    |               |
+
+### Outputs
+No outputs defined
+
+### Usage
+```yaml
+name: Prepare deployment
+
+on:
+  workflow_dispatch:
+jobs:
+  validate_commit:
+    name: Prepare schemas output for deployment
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Get project version
+        uses: SneaksAndData/github-actions/generate_version@0.0.9
+        id: version
+      - name: Generate SAS for upload
+        uses: SneaksAndData/github-actions/get_azure_share_sas@0.0.9
+        with:
+          namespace: k8s-namespace
+          claim_name: persistent-volume
+          directory_name: python/${{steps.version.outputs.version}}
+        id: sas
+      - name: Prepare dbt for deployment
+        uses: SneaksAndData/github-actions/prepare_chemas_deployment@0.0.9
+        with:
+          deployment_root: /dbt
+          project_version: ${{ steps.version.outputs.version }}
+          destination: ${{ steps.sas.outputs.authorized_destination }}
+          project_name: dbt_project
+```
+
+# run_azcopy
+Copy data from local directory on build agent to Azure blob container or file share
+
+### Inputs
+| Name             | Description             | Optional | Default value |
+|------------------|:------------------------|----------|---------------|
+| source_directory | Local directory to copy | False    |               |
+| target           | Target Azure file share | False    |               |
+
+### Outputs
+No outputs defined
+
+### Usage
+```yaml
+name: Copy files
+
+on:
+  workflow_dispatch:
+    
+jobs:
+  copy_files:
+    name: Copy files
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Get project version
+        uses: SneaksAndData/github-actions/generate_version@0.0.9
+        id: version
+      - name: Generate SAS for upload
+        uses: SneaksAndData/github-actions/get_azure_share_sas@0.0.9
+        with:
+          namespace: k8s-namespace
+          claim_name: persistent-volume
+          directory_name: python/${{steps.version.outputs.version}}
+        id: sas
+      - name: Prepare dbt for deployment
+        uses: SneaksAndData/github-actions/prepare_chemas_deployment@0.0.9
+        with:
+          deployment_root: /dbt
+          project_version: ${{ steps.version.outputs.version }}
+          destination: ${{ steps.sas.outputs.authorized_destination }}
+          project_name: dbt_project
+      - name: Deploy project
+        uses: SneaksAndData/github-actions/run_azcopy@0.0.9
+        with:
+          source_directory: /dbt/dbt_project/${{ steps.version.outputs.version }}/*
+          target: ${{ steps.sas.outputs.authorized_destination }}
```

Comment:
this contradicts the description. I think action should be more generic in the usage example